packages/web-react/src/components/Header/README.md

```diff
@@ -4,398 +4,541 @@ The Header is a highly variable and customizable component. It comes in several
-| `UNSAFE_style`     | `CSSProperties`                                | —       | ✕        | HeaderDialog custom style       |
+| `dialogId`         | `string`                                       | —       | ✔        | ID of the linked HeaderDialog   |
+| `isOpen`           | `bool`                                         | `false` | ✔        | Dialog open state               |
+| `menuToggleLabel`  | `string`                                       | `Menu`  | ✕        | Label of the menu toggle button |
+| `onOpen`           | `(event: ClickEvent or KeyboardEvent) => void` | —       | ✔        | Callback for dialog when opened |
+| `UNSAFE_className` | `string`                                       | —       | ✕        | Custom class name               |
+| `UNSAFE_style`     | `CSSProperties`                                | —       | ✕        | Custom style                    |
 
-The rest of the properties are based on the type of `<dialog>`. [Docs][DialogElementDocs]
+The component implements the [`HTMLElement`][mdn-api-html-element] interface.
 
-## HeaderDialogActions
+### Desktop-Only Actions
 
-### API
+As the name suggests, desktop-only actions are intended to display only on
+desktop screens. There are two slots to use: primary actions (aligned to the
+left in LTR documents) and secondary actions (aligned to the right).
 
-| Name               | Type                       | Default | Required | Description                           |
-| ------------------ | -------------------------- | ------- | -------- | ------------------------------------- |
-| `children`         | `ReactNode`                | —       | ✕        | Children node                         |
-| `color`            | [`primary` \| `secondary`] | —       | ✕        | HeaderDialogActions background color  |
-| `UNSAFE_className` | `string`                   | —       | ✕        | HeaderDialogActions custom class name |
-| `UNSAFE_style`     | `CSSProperties`            | —       | ✕        | HeaderDialogActions custom style      |
+👉 It is critical to **make sure all your actions fit the Header on the
+desktop breakpoint**. Spirit intentionally does not provide any overflow
+control here.
 
-The rest of the properties are based on the type of `HTMLElement`. [Docs][HTMLElementDocs]
+```jsx
+<HeaderDesktopActions aria-label="Main navigation">
+  {/* Desktop-only primary actions */}
+</HeaderDesktopActions>
+<HeaderDesktopActions color="secondary">
+  {/* Desktop-only secondary actions */}
+</HeaderDesktopActions>
+```
 
-## HeaderDialogButton
+#### API
 
-### API
+| Name               | Type                       | Default   | Required | Description       |
+| ------------------ | -------------------------- | --------- | -------- | ----------------- |
+| `children`         | `ReactNode`                | —         | ✕        | Children node     |
+| `color`            | [`primary` \| `secondary`] | `primary` | ✕        | Color variant     |
```

Comment:
> Color variant

Maybe we can extend this to cover more stuff it actually changes?

Reply:
I filed https://jira.lmc.cz/browse/DS-1059.